src/Controller/StatisticController.php

```diff
@@ -15,10 +15,21 @@
 include_once __DIR__ . "/../../public/conges/class.conges.php";
 include_once __DIR__ . "/../../public/include/function.php";
 require_once __DIR__ . "/../../public/include/db.php";
+require_once __DIR__ . "/../../public/statistiques/class.statistiques.php";
 include_once __DIR__ . '/../../public/absences/class.absences.php';
+include_once __DIR__ . '/../../public/postes/class.postes.php';
```

Comment:
Why did you add class statistiques and postes ? There are not used in the controller

Reply:
Both classes are used in other methods of the controller. I forgot to delete them temporarily, my bad.